torch/csrc/jit/fuser/interface.cpp

```diff
@@ -63,6 +64,40 @@ std::vector<at::Tensor> debugLaunchGraph(
   return fmap(stack, [](const IValue& iv) { return iv.toTensor(); });
 }
 
+std::string debugGetFusedKernelCode(
+    Graph& graph,
+    at::ArrayRef<at::Tensor> inputs) {
+  // Creates a fusion group node
+  auto wrapper_graph = std::make_shared<Graph>();
+  Node* fusion_group =
+      wrapper_graph->insertNode(wrapper_graph->createFusionGroup());
+  fusion_group->g_(attr::Subgraph, graph.copy());
+  for (size_t i = 0; i < graph.inputs().size(); ++i) {
+    fusion_group->addInput(wrapper_graph->addInput());
```

Comment:
It seems the fusion group input/output types are not being set. Does that not affect the output?

`subgraph_utils` may be useful here.

Reply:
i just copied and pasted this from above, so idk :p